tests/unit/tasks/server/express-server-test.js

```diff
@@ -692,7 +692,7 @@ describe('express-server', function() {
 
       it('calls serverMiddleware on the addons on start', function() {
         return subject.start({
-          host:  '0.0.0.0',
+          host: undefined,
```

Comment:
Question, should we just remove this property entirely? Or do you feel specifying `undefined` explicitly provides value.